test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -37,117 +86,81 @@ define([
             Jupyter.narrative = null;
         });
 
-        beforeEach(function () {
+        beforeEach(async () => {
             node = document.createElement('div');
             document.getElementsByTagName('body')[0].appendChild(node);
 
-            parameterSpec = {
-                data: {
-                    constraints: {
-                        required: false,
-                        options: undefined,
-                        min_length: 1,
-                        max_length: 10000
-                    },
-                    defaultValue: '',
-                    nullValue: '',
-                    sequence: false,
-                    type: 'string'
-                },
-                id: 'fastq_fwd_staging_file_name',
-                multipleItems: false,
-                original: {
-                    advanced: 0,
-                    allow_multiple: 0,
-                    default_values: [''],
-                    description: 'Valid file extensions for FASTA: .fasta, .fna, .fa   Valid file extensions for FASTQ: .fastq, .fnq, .fq; Compressed files (containing files with vaild extentions): .zip, .gz, .bz2, .tar.gz, .tar.bz2',
-                    disabled: 0,
-                    dynamic_dropdown_options: {
-                        data_source: 'ftp_staging',
-                        multiselection: 0,
-                        query_on_empty_input: 1,
-                        result_array_index: 0,
-                        service_params: null
-                    },
-                    field_type: 'dynamic_dropdown',
-                    id: 'fastq_fwd_staging_file_name',
-                    optional: 1,
-                    short_hint: 'Short read file containing a paired end library in FASTA/FASTQ format',
-                    ui_class: 'parameter',
-                    ui_name: 'Forward/Left FASTA/FASTQ File Path'
-                },
-                ui: {
-                    advanced: false,
-                    class: 'parameter',
-                    control: 'dynamic_dropdown',
-                    description: 'Valid file extensions for FASTA: .fasta, .fna, .fa   Valid file extensions for FASTQ: .fastq, .fnq, .fq; Compressed files (containing files with vaild extentions): .zip, .gz, .bz2, .tar.gz, .tar.bz2',
-                    hint: 'Short read file containing a paired end library in FASTA/FASTQ format',
-                    label: 'Forward/Left FASTA/FASTQ File Path',
-                    type: 'dynamic_dropdown'
-                }
-            };
-
-            paramResolver = ParamResolver.make();
-
-            inputControlFacotry = paramResolver.loadInputControl(parameterSpec);
```

Comment:
this is a promise, so the test was failing as we were trying to make the widget before the input control was finished being created